Readme.md

```diff
@@ -7,34 +7,37 @@
 
 ## Introduction
 
-Genio is an IDE for the [Haiku operating system](https://www.haiku-os.org).
-Despite being in an early stage, Genio can already be used to develop and build applications.
+Genio is a native IDE for the [Haiku operating system](https://www.haiku-os.org) 
+Genio can already be used to develop and build applications.
```

Comment:
I'm for removing this, maybe let's just emphasize it in the previous sentence like Genio is a native and fully functional IDE for the haiku..